packages/react-native-renderer/src/ReactNativeSetNativeProps.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
Since you’re going to add more of these, can we name it something more general so we don’t need a file per method?